plugins/chat/assets/javascripts/discourse/services/chat.js

```diff
@@ -278,19 +278,29 @@ export default class Chat extends Service {
       : 0;
   }
 
-  switchChannelUpOrDown(direction) {
+  switchChannelUpOrDown(direction, unreadOnly = false) {
     const { activeChannel } = this;
     if (!activeChannel) {
       return; // Chat isn't open. Return and do nothing!
     }
 
+    let channels, DMs;
```

Comment:
I think DMs should probably be camel case, feels a little weird but I guess because it's an acronym it makes sense. Maybe `directMessages` is better.